public/app/features/search/service/frontend.ts

```diff
@@ -0,0 +1,102 @@
+import { DataFrameView, SelectableValue, ArrayVector } from '@grafana/data';
+import { TermCount } from 'app/core/components/TagFilter/TagFilter';
+
+import { DashboardQueryResult, GrafanaSearcher, QueryResponse, SearchQuery } from '.';
+
+export class FrontendSearcher implements GrafanaSearcher {
```

Comment:
im getting a weird result for empty search:
<img width="979" alt="image" src="https://user-images.githubusercontent.com/23451458/191380834-7c9311ef-f39f-4d7a-b942-be13695dc9fe.png">


the `*` is probably added somewhere as a fallback

Reply:
dooh -- yes